app/models/manageiq/providers/lenovo/physical_infra_manager/refresher.rb

```diff
@@ -4,7 +4,7 @@ def parse_legacy_inventory(ems)
       log_header = "MIQ_LENOVO(#{self.class.name}.#{__method__} Calling for [#{ems.name}])"
       $log.info("#{log_header}")
 
-      ManageIQ::Providers::Lenovo::PhysicalInfraManager::RefreshParser.ems_inv_to_hashes(ems, refresher_options)
+      ManageIQ::Providers::Lenovo::PhysicalInfraManager::RefreshParser.new(ems).ems_inv_to_hashes
```

Comment:
All of the other providers follow this pattern, it might not be strictly necessary here but it is better to keep things consistent.

Reply:
@agrare What pattern do you want me to keep? passing two variables, or using a class method instead of instance method?

Reply:
Both :smile: